src/StoreApi/docs/cart.md

```diff
@@ -332,6 +332,8 @@ curl --header "Nonce: 12345" --request POST https://example-store.com/wp-json/wc
 
 Returns the full [Cart Response](#cart-response) on success, or an [Error Response](#error-response) on failure.
 
+If you want to add filter or supplemental cart item data, before it is passed into `CartController::add_to_cart` use the `woocommerce_store_api_add_to_cart_data` filter, documented on the [Filters page](https://github.com/woocommerce/woocommerce-blocks/blob/4d1c295a2bace9a4f6397cfd5469db31083d477a/docs/third-party-developers/extensibility/hooks/filters.md).
```

Comment:
I wonder if we should link `woocommerce_store_api_add_to_cart_data` directly to the corresponding section. This way, the docs are mire compact, easier to read and guide the developer directly to the necessary section.

```suggestion
If you want to add filter or supplemental cart item data, before it is passed into `CartController::add_to_cart` use the [`woocommerce_store_api_add_to_cart_data`](https://github.com/woocommerce/woocommerce-blocks/blob/4d1c295a2bace9a4f6397cfd5469db31083d477a/docs/third-party-developers/extensibility/hooks/filters.md#woocommerce_store_api_add_to_cart_data) filter.
```